theia-docker/latest.package.json

```diff
@@ -12,32 +12,97 @@
     },
     "dependencies": {
         "@theia/callhierarchy": "latest",
+        "@theia/cli": "latest",
```

Comment:
`@theia/cli` should always go to dev dependecy

Reply:
I understand but what should we do in such a case where we build the `—production` flag. It means that I will not have the `download:plugins` script for such applications.

Reply:
This script should be called during the build time, not at runtime. At the build time `@theia/cli` is present.